src/index/coinstatsindex.cpp

```diff
@@ -228,10 +228,7 @@ bool CoinStatsIndex::WriteBlock(const CBlock& block, const CBlockIndex* pindex)
     m_muhash.Finalize(out);
     value.second.muhash = out;
 
-    CDBBatch batch(*m_db);
```

Comment:
In commit "index: Commit DB_MUHASH and DB_BEST_BLOCK to disk together" (010156c07f28e2c8dc30d4408942d1e1d88ab54d)

I might add a code comment here like "Intentionally do not update DB_MUHASH here so it stays in sync with DB_BEST_BLOCK, and index is not corrupted if there is an unclean shutdown"

Reply:
Thanks, I added a comment here.